### DISCUSSION TITLE
bug: clearjumps on VimEnter doesn't take affect

### DISCUSSION THREAD

**Discussion by leaxoy** [LazyVim/LazyVim#3586] [Q&A]
### Did you check docs and existing issues?

- [X] I have read all the LazyVim docs
- [X] I have searched the existing issues of LazyVim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

NVIM v0.11.0-dev-212+gd8e384b7b-Homebrew

### Operating system/version

MacOs 14.5

### Describe the bug

`autocmd("VimEnter", {command = "clearjumps"}) ` doesn't take effect.

### Steps To Reproduce

set autocmd

### Expected Behavior

clear all jumps

### Repro

```Lua
-- DO NOT change the paths and don't remove the colorscheme
local root = vim.fn.fnamemodify("./.repro", ":p")

-- set stdpaths to use .repro
for _, name in ipairs({ "config", "data", "state", "cache" }) do
  vim.env[("XDG_%s_HOME"):format(name:upper())] = root .. "/" .. name
end

-- bootstrap lazy
local lazypath = root .. "/plugins/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({ "git", "clone", "--filter=blob:none", "https://github.com/folke/lazy.nvim.git", lazypath, })
end
vim.opt.runtimepath:prepend(lazypath)

-- install plugins
local plugins = {
  "folke/tokyonight.nvim",
  "folke/LazyVim",
  -- add any other plugins here
}
require("lazy").setup(plugins, {
  root = root .. "/plugins",
})

vim.cmd.colorscheme("tokyonight")
-- add anything else here
```

**Comment by dpetka2001** [comment]
Have you tried putting this autocmd in your `/lua/config/options.lua` to see if it works?

**Reply by leaxoy** [reply]
Finally I put it to my init.lua in root dir then it works.😀
